hack/update-staging-godeps.sh

```diff
@@ -71,7 +71,9 @@ function updateGodepManifest() {
     go run "${KUBE_ROOT}/staging/godeps-json-updater.go" --godeps-file="${TMP_GOPATH}/src/k8s.io/${repo}/Godeps/Godeps.json" --client-go-import-path="k8s.io/${repo}"
 
     # commit so that following repos do not see this repo as dirty
-    git add vendor >/dev/null
+    # pipe through true so that command script won't error if there are no
+    # files to add
+    git add vendor/ >/dev/null || true
```

Comment:
I'm running git 2.12.0 on OS X, and it does fail with exit code 128.

```
Rewriting Godeps.json to remove commits that don't really exist because we haven't pushed the prereqs yet
fatal: pathspec 'vendor' did not match any files
!!! [0312 10:21:56] Call tree:
!!! [0312 10:21:56]  1: hack/update-staging-godeps.sh:99 updateGodepManifest(...)
!!! Error in hack/update-staging-godeps.sh:63
  Error in hack/update-staging-godeps.sh:63. '((i<3-1))' exited with status 128
Call stack:
  1: hack/update-staging-godeps.sh:63 updateGodepManifest(...)
  2: hack/update-staging-godeps.sh:99 main(...)
Exiting with status 1
```